package/installation.suse-manager-server.xsl

```diff
@@ -178,10 +178,10 @@ textdomain="control"
                 <proposed_configurable config:type="boolean">true</proposed_configurable>
 		<proposed config:type="boolean">true</proposed>
 
-                <desired_size config:type="disksize">2 GiB</desired_size>
+                <desired_size config:type="disksize">4 GiB</desired_size>
                 <min_size config:type="disksize">2 GiB</min_size>
                 <max_size config:type="disksize">unlimited</max_size>
-                <max_size_lvm config:type="disksize">40 GiB</max_size_lvm>
+                <max_size_lvm config:type="disksize">4 GiB</max_size_lvm>
```

Comment:
Just to be sure, really 10x  less?

Reply:
Yes, I expect all of the values to need more tuning anyway